tests/benchmark/runner/src/nonKeyed.ts

```diff
@@ -0,0 +1,179 @@
+import * as chrome from 'selenium-webdriver/chrome';
```

Comment:
don't think we need this (just checks that the implementation is keyed or not I think)

Reply:
Removed